data/categories.txt

```diff
@@ -2086,35 +2438,120 @@ ro:Cimitir
 nb:Gravplass
 fi:Hautausmaa
 
-amenity-hospital|amenity-doctors|amenity-dentist|amenity-clinic
-en:3Hospital|clinic|3doctor|U+1F691|U+1F3E5|U+1F489|U+1F48A
-ru:3Больница|3госпиталь|врач|доктор
-uk:3Лікарня|клініка|лікар|травмпункт
-de:3Krankenhaus|Spital|3Arzt|Doktor|medizinische Einrichtung|Gesundheitswesen
-fr:3Hôpital|clinique|3médecin|3docteur
-it:3Ospedale
-es:3Hospital
-ko:1병원|병원|의사|의사가
-ja:1病院|クリニック|医師|医者|ドクター|救急|診療
-cs:3Nemocnice|klinika|pohotovost|zdravotnické centrum|3lékař|doktor
-sk:3Nemocnica|klinika|pohotovosť|zdravotnícke zariadenie|3lekár|doktor
-nl:3Ziekenhuis|kliniek|hospitaal|tandarts|3dokter
+amenity-hospital
+en:4Hospital|clinic|3doctor|U+1F691|U+1F3E5|U+1F489|U+1F48A
+ru:4Больница|4госпиталь|врач|доктор
+uk:4Лікарня|клініка|лікар|травмпункт
+de:4Krankenhaus|Spital|3Arzt|Doktor|medizinische Einrichtung|Gesundheitswesen
+fr:4Hôpital|clinique|3médecin|3docteur
+it:4Ospedale
+es:4Hospital
+ko:병원|병원|의사|의사가
+ja:病院|クリニック|医師|医者|ドクター|救急|診療
+cs:4Nemocnice|klinika|pohotovost|zdravotnické centrum|3lékař|doktor
+sk:4Nemocnica|klinika|pohotovosť|zdravotnícke zariadenie|3lekár|doktor
+nl:4Ziekenhuis|kliniek|hospitaal|tandarts|3dokter
 zh-Hant:1醫院|醫生|診所|醫療|診療
-pl:3Szpital|klinika|lekarz|doktor|leczenie
-pt:3Clínica|clínica|3médico
-hu:3Kórház|klinika|3orvos
+pl:4Szpital|klinika|lekarz|doktor|leczenie
+pt:4Clínica|clínica|3médico
+hu:4Kórház|klinika|3orvos
 th:3คลินิก|แพทย์
-zh-Hans:1诊所|医生|医院
+zh-Hans:诊所|医生|医院
 ar:المستشفى|عيادة
-da:3Hospital|3klinik
-tr:3Hastane|3klinik|doktor
-sv:3Sjukhus|klinik|3läkare
+da:4Hospital|klinik
+tr:4Hastane|3klinik|doktor
+sv:4Sjukhus|klinik|3läkare
 vi:Bệnh viện
 id:Rumah sakit
 ro:Spital
 nb:Sykehus
 fi:Sairaala
 
+amenity-clinic
+en:Clinic|1hospital|U+1F3E5|U+1F489
+ar:عيادة|مستشفى
+cs:Klinika|nemocnice
+da:Klinik|hospital
+de:Krankenhaus|Klinik
+es:Clínica|hospital
+fi:Klinikka|sairaala
+fr:Clinique|hôpital
+hu:Klinika|kórház|szanatórium
+id:Klinik|rumah sakit
+it:Clinica|ospedale
+ja:クリニック|病院、診療所
+ko:병원|의원
+nl:Kliniek|ziekenhuis
+nb:Klinikk|sykehus
+pl:Szpital
+pt:Clínica|centro de saúde
+ro:Clinică|spital
+ru:Поликлиника|клиника
+sk:Poliklinika|nemocnica|zdravotné stredisko
+sv:Klinik|sjukhus
+th:คลินิก|โรงพยาบาล
+tr:Klinik
+uk:Поліклініка|клініка
+vi:Bệnh viện|BV
+zh-Hans:诊所
+zh-Hant:診所
+
+amenity-doctors
+en:Doctor|U+1F489|U+1F3E5|U+1F691
+ar:مكتب طبيب
+cs:Lékařská ordinace
+da:Lægekontor
+de:Arztpraxis
+es:Consulta del médico
+fi:Lääkärin vastaanotto
+fr:Cabinet médical
+hu:Rendelő|orvosi rendelő
+id:Praktik dokter
+it:Studio medico|ambulatorio
+ja:医者のオフィス
+ko:개인병원
+nl:Huisartsenpost|huisarts
+nb:Legekontor|legesenter
+pt:Gabinete do médico
+ro:Cabinet medical
+ru:Врач
+sk:Ambulancia
+sv:Läkarmottagning
+th:ห้องแพทย์
+tr:Doktor muayenehanesi
+uk:Лікар
+vi:Phòng khám
+zh-Hans:医师办公室
+zh-Hant:醫師診間
```

Comment:
Будем, сейчас сделаю. И на клиники тоже.

> On Mar 15, 2016, at 14:12, Viktor Govako <notifications@github.com> wrote:
> 
> In data/categories.txt <https://github.com/mapsme/omim/pull/2195#discussion_r56146965>:
> 
> > +it:Studio medico|ambulatorio
> > +ja:医者のオフィス
> > +ko:개인병원
> > +nl:Huisartsenpost|huisarts
> > +nb:Legekontor|legesenter
> > +pt:Gabinete do médico
> > +ro:Cabinet medical
> > +ru:Врач
> > +sk:Ambulancia
> > +sv:Läkarmottagning
> > +th:ห้องแพทย์
> > +tr:Doktor muayenehanesi
> > +uk:Лікар
> > +vi:Phòng khám
> > +zh-Hans:医师办公室
> > +zh-Hant:醫師診間
> Вроде как hospital - у нас называется дженерик категория со стартового экрана.
> amenity-doctors на нее не будем матчить?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/mapsme/omim/pull/2195/files#r56146965 <https://github.com/mapsme/omim/pull/2195/files#r56146965>